README.md

```diff
@@ -1,2 +1,11 @@
-# beekeeper-terraform
-Terraform deployment scripts for Beekeeper
+# Overview
+Terraform deployment scripts for [Beekeeper](https://github.com/ExpediaGroup/beekeeper).
+
+# Modules
+* [Main](https://github.com/ExpediaGroup/beekeeper/tree/master/main) - main AWS components required to run Beekepeer e.g. ECS, SQS, IAM etc.
+* [Lambda](https://github.com/ExpediaGroup/beekeeper/tree/master/lambda) - slack notifying Lambda which monitors Beekeeper's dead letter queue
```

Comment:
This is optional right? If I don't use slack I can just skip this module and everything will work except alerts

Reply:
Yep, it's a totally separate module and doesn't need to be used. I'll add that to the Readme to make it more clear.